recipes/ibis-framework/meta.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: ibis-framework
+  version: "0.7.1"
+
+source:
+  git_url: https://github.com/cloudera/ibis
+  git_rev: v0.7.1
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest
+    - numpy >=1.7.0
+    - pandas >=0.12.0
+    - impyla >=0.13.6
+    - python-hdfs >=2.0.0
+    - sqlalchemy >=1.0.0
+    - six
+    - mock  # [not py35]
```

Comment:
Is this because `mock` in Python 3.5 is sufficient? If so, could we please try `# [py<35]`? It will be less work when Python 3.6 comes out. :smile: